extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/tunnel/RestrictedGuacamoleTunnelService.java

```diff
@@ -180,8 +187,24 @@ protected ModeledConnection acquire(RemoteAuthenticatedUser user,
             @Override
             public int compare(ModeledConnection a, ModeledConnection b) {
 
-                return getActiveConnections(a).size()
-                     - getActiveConnections(b).size();
+                // Active connections
+                int Ca = getActiveConnections(a).size();
```

Comment:
Both the lack of camel case and the "Net Weight" came from referencing the following pages:
http://kb.linuxvirtualserver.org/wiki/Weighted_Least-Connection_Scheduling
https://www.citrix.com/blogs/2010/10/01/load-balancing-weights/
https://www.citrix.com/blogs/2010/09/02/load-balancing-least-connections/

(as did the initial, and now-removed, concept of multiplying by the arbitrary 10K number).  Anyway, I don't know that "Net Weight" is actually called out in those, but the Nw abbreviation is used in the Citrix page.  I can go with something else - computed weight or calculated weight - if that makes more sense.